commerce_coordinator/apps/commercetools/tests/test_views.py

```diff
@@ -160,6 +160,18 @@ def test_view_returns_expected_error_no_order(self, mock_customer, mock_order, m
 
         self.assertEqual(response.status_code, 200)
 
+    def test_unauthorized_user(self, mock_customer, mock_order, mock_signal):
+        """Check unauthorized user is forbidden."""
+
+        # Login
+        self.client.login(username=self.test_user_username, password=self.test_password)
+
+        # Send request
+        response = self.client.post(self.url, data=EXAMPLE_COMMERCETOOLS_ORDER_SANCTIONED_MESSAGE, format='json')
```

Comment:
```suggestion
        response = self.client.post(self.url, data=EXAMPLE_COMMERCETOOLS_ORDER_FULFILL_MESSAGE, format='json')
```

Reply:
Applied as 60f53b4df2c4ffcbcc031f35ce977c140d21d25b, thank you!